files/zh-cn/web/api/canvas_api/tutorial/basic_animations/index.md

```diff
@@ -681,6 +681,7 @@ function tmz() {
 
 ##### Output
 
+```html
```

Comment:
This should be a rendered html table in markdown (not a code example).

And I've try to replace this with a `{{EmbedLiveSample}}` macro, but it doesn't work. Also, this `Snake Game` example seems to be removed in `en-US`. Maybe we'd better to remove this part (from line 453 to line 701)